config/v1/types_ingress.go

```diff
@@ -31,6 +31,16 @@ type IngressSpec struct {
 	//
 	// Once set, changing domain is not currently supported.
 	Domain string `json:"domain"`
+
+	// appsDomain is an optional domain to use instead of the one specified
+	// in the domain field when a Route is created without specifying an explicit
+	// host. If appsDomain is nonempty, this value is used to generate default
```

Comment:
If `appsDomain` is nonempty what does "this value" refer to?

Reply:
That nonempty value of `appsDomain` is used to generate the default host.

Reply:
It basically means that the value of `appsDomain` will be used instead of `domain` for routes that do not have a host specified.

Reply:
thanks; if it's not too nitpicky (or to late) the longer form reads better than "this value" but I would not hold the PR for this though.